float32[] radian
---
bool success